projects/1.16/assets/snail-mail/snailmail/lang/zh_cn.json

```
@@ -0,0 +1,46 @@
+{
+    "itemGroup.snailmail.main": "蜗牛信箱",
+    "entity.snailmail.snail_man": "牛递员",
+    "container.snailmail.snail_box": "蜗牛信箱",
+    "container.snailmail.envelope": "信封",
+    "container.snailmail.envelope.textfield.to": "收信地",
+    "container.snailmail.envelope.textfield.from": "发信地",
+    "block.snailmail.snail_box": "蜗牛信箱",
+    "item.snailmail.envelope_open": "信封 (已拆)",
+    "item.snailmail.envelope_closed": "信封 (未拆)",
+    "item.snailmail.stamp": "邮票",
+    "tooltip.snailmail.to": "送往: %s",
+    "tooltip.snailmail.from": "来自: %s",
+    "tooltip.snailmail.stamped": "已盖邮戳",
+    "tooltip.snailmail.delivery_failed": "递送失败",
+    "modal.snailmail.no_envelope": "服务器上的插槽中没有信封...可能发生了错误?",
+    "modal.snailmail.no_stamp": "信封还未盖上邮戳!",
+    "modal.snailmail.no_address": "信封没有地址!",
+    "modal.snailmail.invalid_address": "找不到与该地址名称匹配的用户: %s",
+    "modal.snailmail.no_boxes": "%s 没有任何蜗牛邮箱",
+    "modal.snailmail.box_no_exist": "所选信箱不再存在或成员已更改。发送失败。",
+    "modal.snailmail.select_box": "选择要将邮件发送到的信箱",
+    "modal.snailmail.located": "位于: %s in %s",
+    "modal.snailmail.no_location": "私人地址",
+    "modal.snailmail.sending": "发送",
+    "modal.snailmail.select": "选择",
+    "modal.snailmail.close": "结束",
+    "modal.snailmail.success": "成功! 已派出牛递员运送你的包裹.",
+    "modal.snailmail.failed_to_add": "未能将成员添加到列表。该用户可能无效或以前未登录。",
+    "modal.snailmail.remove_selected": "删除选定项",
+    "modal.snailmail.add": "添加成员",
+    "modal.snailmail.done": "完成",
+    "modal.snailmail.member": "成员",
+    "modal.snailmail.owner": "拥有者",
+    "container.snailmail.snail_box.members": "成员组",
+    "container.snailmail.snail_box.public": "显示地址",
+    "message.snailmail.noperm": "您无权打开此邮箱。",
+    "container.snailmail.snail_box.send": "发送",
+    "container.snailmail.snail_box.textfield.name": "邮箱名称",
+    "container.snailmail.snail_box.open_envelope": "拆开信件",
+    "config.snailmail": "Snail Mail 设置",

```

```suggestion
    "config.snailmail": "蜗牛邮政设置",
```
注意一下这边需要和上面最后敲定的译名同步，不一定就是这个（）